### PR TITLE
requirements.txt installation

### PR DESCRIPTION
**Pull request by iamtekson**
The installation of `requirements.txt` (python dependencies) is added

### CODE REVIEW

File: install/advanced/core/index.rst [hunk 1]
```diff
@@ -181,6 +181,8 @@ At this point your command prompt shows a ``(geonode)`` prefix, this indicates t
   # Clone the GeoNode source code on /opt/geonode
   cd /opt
   git clone https://github.com/GeoNode/geonode.git geonode
+  cd /opt/geonode
+  pip install -r requirements.txt
```
Comment (afabiani) [inline install/advanced/core/index.rst:185]: Already covered a few lines below

https://github.com/GeoNode/documentation/pull/54/files#diff-e59499c486837b30526c50c0777516cfL204-L206

**Review by afabiani — CHANGES_REQUESTED** [review]
Already covered a few lines below

https://github.com/GeoNode/documentation/pull/54/files#diff-e59499c486837b30526c50c0777516cfL204-L206

File: install/advanced/core/index.rst (hunk 1 above)
Comment (iamtekson) [reply install/advanced/core/index.rst:185]: I don't know why but it is still not showing in the documentation, that's why I added this section. Anyway, it is already covered in a few lines below, you can close this pull request. 
image

Comment (afabiani) [reply install/advanced/core/index.rst:185]: Oh, I see. You are right. Probably a new line missing before the next code section.

Let me close this and send a correct patch for this issue.

Thanks for your feedkack.

Comment (afabiani) [reply install/advanced/core/index.rst:185]: Here is the fix

https://github.com/GeoNode/documentation/commit/7d6738909c588e6756baae1aac89e9c36ea35af7

It should be updated in few minutes

**Pull request closed without merging**